src/twisted/conch/ssh/keys.py

```diff
@@ -390,6 +465,17 @@ def _fromString_PRIVATE_OPENSSH(cls, data, passphrase):
                     )
                 ).private_key(backend=default_backend())
             )
+        elif kind == b'EC':
+            newKey = cls(load_pem_private_key(data, passphrase, default_backend()))
+
+            curve = _oidTable[str(decodedKey[2]).encode('utf-8')]
+            # Reverse look up the nist curve name to be referenced later
+            for k,v in _curveTable.items():
```

Comment:
Just a coding-standard nitpick: there should be a space after the comma in this line.  Sorry the automatic checker didn't pick that up.